lang/en.json

```diff
@@ -39,7 +39,7 @@
   "Dashboard.Overview.ChooseWebsites": "Choose websites",
   "Dashboard.Running.Running": "Running:",
   "Dashboard.Running.EstimatedTimeLeft": "Estimated time left:",
-  "Dashboard.Running.Seconds": "{seconds} seconds",
+  "Dashboard.Running.Seconds": "{seconds}",
```

Comment:
@sarathms can you update this in the places where it should be updated too?

Reply:
We need not translate this string at all. Instead we can drop using that this key. It leads to another problem that I mentioned in the other comment.